tyler-tcdittman/hw_1/bottlesOfBeer.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
+  If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
+
+  Be sure that all of your output includes proper spacing (no words or values running into each other).
+
+  Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
+
+  For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
+*/
+
+
+for (var number = 99; number >= 0; number--) {
+  if (number > 2) {
+    console.log(number + " " + "bottles of beer on the wall," + " " + number + " " + "bottles of beer.\nTake one down and pass it around," + " " + (number - 1) + " " + "bottles of beer on the wall.");
+  } else if (number > 1) {
+    console.log(number + " " + "bottles of beer on the wall," + " " + number + " " + "bottles of beer.\nTake one down and pass it around," + " " + (number - 1) + " " + "bottle of beer on the wall.");
+  } else if (number > 0) {
+    console.log(number + " " + "bottle of beer on the wall," + " " + number + " " + "bottle of beer.\nTake one down and pass it around," + " " + (number - 1) + " " + "bottles of beer on the wall.");
+  } else {
+    console.log("No more bottles of beer on the wall, no more bottles of beer.\nGo to the store any buy some more, 99 bottles of beer on the wall.");
+  }
+};
+
+
+/*
+  Initially I wrote line 17 as
+
+  } else if (number = 2) {
+
+  But the loop repeated infinitey at number = 2. Why did it do that and not increment to number = 1?
```

Comment:
one equal sign means I'm setting the variable. I needed three to use the "is exactly equal to" operator. duh! thank you!!